### ISSUE TITLE
Dragonflight Errors

### ISSUE THREAD

**Issue by arcadiatostark** [ketho-wow/MaxCam#1]
```
2x FrameXML\Bindings.xml:1 Interface/AddOns/MaxCam/MaxCam.lua:1 (null)

4x Interface/AddOns/MaxCam/MaxCam.lua:1 (null)

7x Interface/AddOns/MaxCam/MaxCam.lua:160: hooksecurefunc(): InterfaceOptionsList_DisplayPanel is not a function
[string "=[C]"]: in function `hooksecurefunc'
[string "@Interface/AddOns/MaxCam/MaxCam.lua"]:160: in main chunk

Locals:
```

**Comment by CRYPTfromCATACOMBZ** [comment]
Seems abandoned :/

**Comment by Ketho** [comment]
I'm just terribly backlogged :(

**Comment by Ketho** [comment]
Should be fixed in [v2.6.1](https://github.com/ketho-wow/MaxCam/releases/tag/v2.6.1)
